### ISSUE TITLE
Module not found

### ISSUE THREAD

**Issue by Kikketer** [francisbesset/font-awesome-sass-loader#2]
Not sure if this is a webpack error or what, but this is the first time I've come across this error.  When I add
`require('font-awesome-sass-loader');`
to the top of my webpack.config.js file, I get the following error:

```
Cannot find module 'style!css!sass!./font-awesome-sass-styles!./font-awesome-sass.config.js'
```

It's not entirely helpful, am I missing "style", "css", "sass" or something with "font-awesome-sass-styles".  I've also attempted to copy and paste the configs from your project into the root (next to eachother) and use the 
`require('font-awesome-sass-loader!./font-awesome-sass.config.js');`
format.

Still came back with the module not found, but described the exact require line.

My webpack config has been using SCSS and fonts already:

```
      {
        test: /\.(woff|woff2)$/,
        loader: 'url-loader?limit=10000&mimetype=application/font-woff&name=fonts/[name].[ext]'
      },
      {test: /\.ttf$/, loader: 'url-loader?limit=10000&mimetype=application/octet-stream&name=fonts/[name].[ext]'},
      {test: /\.eot$/, loader: 'file-loader'},
```

Like I said, this is the first time I've had this error, and I'm not entirely sure where the problem lies.

**Comment by asabirov** [comment]
npm install --save font-awesome font-awesome-sass-loader

**Comment by borysn** [comment]
I'm having a similar issue atm. Seems the font-awesome-loader isn't finding the file `font-awesome-sass-styles.loader.js` but it's there. I think this might be an issue with the webpack 2.0 beta config

`ERROR in ./~/font-awesome-sass-loader/index.js
Module not found: Error: Can't resolve './font-awesome-sass-styles' in '/home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader'
resolve './font-awesome-sass-styles' in '/home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader'
  using description file: /home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/package.json (relative path: .)
  after using description file: /home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/package.json (relative path: .)
    using description file: /home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/package.json (relative path: ./font-awesome-sass-styles)
      as directory
        /home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/font-awesome-sass-styles doesn't exist
      no extension
        /home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/font-awesome-sass-styles doesn't exist
      .js
        /home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/font-awesome-sass-styles.js doesn't exist
      .json
        /home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/font-awesome-sass-styles.json doesn't exist
[/home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/font-awesome-sass-styles]
[/home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/font-awesome-sass-styles]
[/home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/font-awesome-sass-styles.js]
[/home/p3pt/dev/java/projects/spring-boot-angular2/master/sba2-wip/src/main/web/node_modules/font-awesome-sass-loader/font-awesome-sass-styles.json]
 @ ./~/font-awesome-sass-loader/index.js 1:0-82
 @ ./app/vendor.ts`

**Comment by borysn** [comment]
I'm going to fork and submit a pull request for a fix, let me know if it works for you

**Comment by francisbesset** [comment]
Fix by PR #3
